src/remote.ts

```diff
@@ -386,7 +388,27 @@ export class Remote {
       // Probably just doesn't exist!
       sshConfigRaw = ""
     }
-    const parsedConfig = SSHConfig.parse(sshConfigRaw)
+
+    // We are going to extract only the coder config
+    let coderConfigRaw = ""
+    const startOfCoderConfig = sshConfigRaw.indexOf(startBlockComment)
+    const endOfCoderConfig = sshConfigRaw.lastIndexOf(endBlockComment)
+
+    // There is an existent coder config
+    if (startOfCoderConfig > -1 && endOfCoderConfig > -1) {
```

Comment:
We should also check `startOfCoderConfig < endOfCoderConfig`. See https://github.com/coder/coder/pull/5859.

Reply:
Good catch!